src/Modules/getTreeDescendants.lua

```diff
@@ -0,0 +1,23 @@
+local flipbook = script:FindFirstAncestor("flipbook")
+
+local types = require(flipbook.types)
+
+local function getTreeDescendants(root: types.ComponentTreeNode): { types.ComponentTreeNode }
+	local descendants: { types.ComponentTreeNode } = {}
+
+	local function traverse(node: types.ComponentTreeNode, isRoot: boolean)
+		if not isRoot then
+			table.insert(descendants, node)
+		end
+
+		for _, child in ipairs(node.children) do
```

Comment:
Luau knows how to handle tables without `pairs` and `ipairs` now. This can be simplified 
```suggestion
		for _, child in node.children do
```

Reply:
Thanks for your suggestion. Just applied the changes.